### PR TITLE
update constants for na45 and na46 instances

### PR DESCRIPTION
**Pull request by arunjayaramareddy**

### CODE REVIEW

File: src/main/java/com/salesforce/cdp/queryservice/core/QueryServiceConnection.java [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
     public QueryServiceConnection(String url, Properties properties) {
         this.properties = properties;
-        this.serviceRootUrl = getServiceRootUrl(url);
+        this.serviceRootUrl = url.substring(Constants.DATASOURCE_TYPE.length());
```
Comment (palaniappa) [inline src/main/java/com/salesforce/cdp/queryservice/core/QueryServiceConnection.java:42]: this will not work if the login url ends with /.

**Review by palaniappa — CHANGES_REQUESTED** [review]

File: src/main/java/com/salesforce/cdp/queryservice/util/Constants.java [hunk 2]
```diff
@@ -46,6 +46,14 @@
     public static final String PROD_DEFAULT_CLIENT_ID = "3MVG9VeAQy5y3BQVJqaUbFmV5jd8imcck2K5idmrTTGocSu9qZZ6qkbuEkxECKVYwmzm3WgvxkujqsxZDcBpL";
     public static final String PROD_DEFAULT_CLIENT_SECRET = "1007FFFBA2B6B6B1EF21E2B03F4C4F692ADE10AB7DEF19E00D8AAF85EF6F6A12";
 
+    public static final String NA45_SERVER_URL = "na45.test1.pc-rnd.salesforce.com";
+    public static final String NA45_DEFAULT_CLIENT_ID = "3MVG9XjhiDAzhaqaC4RR0yon8blsafwWlTnckUT8bEduWr0v9UpiQ2cJkmhNtFI1kVqpY8WpyE9JYkG.ZtgiE";
+    public static final String NA45_DEFAULT_CLIENT_SECRET = "84642974065EDF90CA6F30FFEE23E2C16BDFA84D3083EF90E0AE905FA46131AD";
+
+    public static final String NA46_SERVER_URL = "na46.test1.pc-rnd.salesforce.com";
+    public static final String NA46_DEFAULT_CLIENT_ID = "3MVG9XjhiDAzhaqaC4RR0yon8blsafwWlTnckUT8bEduWr0v9UpiQ2cJkmhNtFI1kVqpY8WpyE9JYkG.ZtgiE";
```
Comment (palaniappa) [inline src/main/java/com/salesforce/cdp/queryservice/util/Constants.java:54]: both na45 and 46 seems to have the same client id and secret.

File: src/main/java/com/salesforce/cdp/queryservice/core/QueryServiceConnection.java (hunk 1 above)
Comment (arunjayaramareddy) [reply src/main/java/com/salesforce/cdp/queryservice/core/QueryServiceConnection.java]: fixed

File: src/main/java/com/salesforce/cdp/queryservice/util/Constants.java (hunk 2 above)
Comment (arunjayaramareddy) [reply src/main/java/com/salesforce/cdp/queryservice/util/Constants.java]: fixed

**Review by palaniappa — APPROVED** [review]

**Pull request merged**